tcms/testruns/tests/test_admin.py

```diff
@@ -91,3 +99,42 @@ def test_changelist_view_icon_column(self):
             """
             ),
         )
+
+    @parameterized.expand([[positive_list], [neutral_list], [negative_list]])
```

Comment:
I believe you don't need to make a list (`[]`) which contains variables of type list. However for each test data we need a human readable name as the first argument. Refer to this example https://stackoverflow.com/a/32939/1431647